src/App.js

```diff
@@ -31,6 +31,16 @@ function App() {
     }
   },[]);
 
+  const validateNotificationUrl = (url) => {
+    let parseURL = url;
+
+    if (!url.includes("//")){
+      parseURL = "http://" + url;
+    }
```

Comment:
Looks like a generic problem in javascript from the discussion in https://stackoverflow.com/questions/3543187/prepending-http-to-a-url-that-doesnt-already-contain-http.

Reply:
shouldn't this be https? We really shouldn't be using http anywhere.

Reply:
Do you prefer a regex allowing both`http` and `https`?

Reply:
I think links from our team are usually safe to just bind `https`